### ISSUE TITLE
rename propoposal's "questions" and "answers" to "concerns" and "clarifications"

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#214]
After watching the vilfredo presentatin on youtube, I strongly propose we rename the "question" and "answer" links within each proposal into a more general and not confusing "concern" (a question or doubt about the proposal) and "clarification" (the answer to the concern).

**Comment by pietrosperoni** [comment]
disagree. A concern is more like a negative comment. We can rename that. But the question about a proposal I think should remain called question. The answer could be called clarification from the author. Also at some point we should have people who claim to understand a proposal very well start to be able to explain it. And if not lose points. And at that point the term clarification start to make even more sense. In all case I don't see the urgency, so @giovanisp I would move this for at least 2.2 if not 2.3 when we design the whole flow experience.

**Comment by giovanisp** [comment]
Don't you see usability problems with people thinking that they have to open a "question" where people can write proposals and each proposal has in turn votes, comments and questions (again) that should be answered (now "clarified").
I want to get rid of the double definition of the meaning of "question". Maybe the main "question" can be called "decision"? Or the proposal question be called "doubts" ? Or.... Vilfredo question for it?

Inviato da iPad

**Comment by pietrosperoni** [comment]
Yeah, very good point! Not sure what the solution is, but calling it concern does not sound correct. Should look into synonyms

**Comment by giovanisp** [comment]
Welcome to https://vilfredo.org/question/23 @vvorski @pietrosperoni @derekpa

**Comment by pietrosperoni** [comment]
wow:
![screen shot 2015-02-12 at 16 21 26](https://cloud.githubusercontent.com/assets/1333674/6170105/6d801e60-b2d2-11e4-8024-148c722af1d9.png)

**Comment by pietrosperoni** [comment]
we are going to have fun in the voting phase ;-)

**Comment by giovanisp** [comment]
https://vilfredo.org/question/23 is in voting phase @vvorski  @pietrosperoni @derekpa  please participate and, also, let me know if anyone else has to be invited.

**Comment by vvorski** [comment]
Concerns has negative connotation.

V/.

**Comment by giovanisp** [comment]
Yes, there are 26 proposals under voting in https://vilfredo.org/question/23 you are invited.
For each proposal you can leave pro/con comments.


> Il giorno 14/feb/2015, alle ore 21:52, vvorski <notifications@github.com> ha scritto:
> 
> Concerns has negative connotation.
> 
> V/.
>

**Comment by giovanisp** [comment]
@vvorski  @derekpa  @pietrosperoni 
I am about to end the voting at https://vilfredo.org/question/23 if you still want to vote...

**Comment by pietrosperoni** [comment]
I would suggest to wait until you have a second generation. This is a difficult question that requires to integrate two different questions/enquires.

**Comment by giovanisp** [comment]
Skipping this release then.
